android_configure.py

```diff
@@ -0,0 +1,76 @@
+import platform
+import sys
+import os
+
+# TODO: In next version, it will be a JSON file listing all the patches, and then it will iterate through to apply them.
+def patch_android():
+    print("- Patches List -")
+    print("[1] [deps/v8/src/trap-handler/trap-handler.h] related to https://github.com/nodejs/node/issues/36287")
+    if platform.system() == "Linux":
+        os.system('patch -f ./deps/v8/src/trap-handler/trap-handler.h < ./android-patches/trap-handler.h.patch')
```

Comment:
Please use `subprocess.run()` instead of `os.system()` for security and compatibility reasons.
* https://docs.python.org/3/library/subprocess.html#subprocess.run